source/_components/notify.google_voice.markdown

```diff
@@ -34,3 +34,18 @@ Configuration variables:
 ### Usage
 
 Google Voice is a notify platform and thus can be controlled by calling the notify service [as described here](/components/notify/). It will send a notification to all devices listed in the notification **target**.
+
+```yaml
+# Example automation notification entry
+  - alias: Send a notification
```

Comment:
Could you add the automation header above this line so people know it's for that component?